.github/workflows/publish.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish Package
+
+on:
+  push:
+    branches:
+      - '!*'
+    tags:
+      - 'v*'
+
+jobs:
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+          submodules: recursive
+
+      - uses: oven-sh/setup-bun@v2
+      - run: bun install --frozen-lockfile
+      - run: bun run build
+      - run: bun publish --access public
+        env:
+          NPM_CONFIG_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Update CHANGELOG
+        id: changelog
+        continue-on-error: true
+        uses: requarks/changelog-action@v1
+        with:
+          token: ${{ github.token }}
+          tag: ${{ github.ref_name }}
+
+      - name: Create Release
+        continue-on-error: true
+        uses: ncipollo/release-action@v1
```

Comment:
This doesn't make sense I believe.
The release should be to NPM, and this is simply publising to githbub releases

Reply:
Yes this is particular dedicated just for release notes.